gatsby/plugins/gatsby-source-bnf/src/graphql-schema/drug.ts

```diff
@@ -144,15 +144,26 @@ export const drugSchema = `
 		slug: String! @slug(field: "form")
 
 		"A list of any cautionary and advisory labels for the medicinal form."
-		cautionaryAndAdvisoryLabels: [String!]
+		cautionaryAndAdvisoryLabels: [${BnfNode.MedicinalFormLabel}]
```

Comment:
This should be a required (non-null) list of non-null objects
```suggestion
		cautionaryAndAdvisoryLabels: [${BnfNode.MedicinalFormLabel}!]!
```
```suggestion
		cautionaryAndAdvisoryLabels: [${BnfNode.MedicinalFormLabel}]
```